index.bs

```diff
@@ -267,8 +279,8 @@ The {{VideoFrame/codedWidth!!attribute}} and
 {{VideoFrame/codedHeight!!attribute}} constitute the production aperture of the
 image.
 
-Instances of {{VideoFrame}} are created with an internal slot <dfn attribute
-for=VideoFrame>\[[frame]]</dfn>.
+Issue: [ttoivone] need to specify in a section whether codedSize or visibleSize
```

Comment:
Link to this issue https://github.com/WICG/web-codecs/issues/26

Reply:
Done